src/components/Envelope.vue

```diff
@@ -127,6 +128,7 @@ export default {
 	methods: {
 		onToggleFlagged() {
 			this.$store.dispatch('toggleEnvelopeFlagged', this.data)
```

Comment:
use .then(...) to await the change on the server. And we should only track this when an unflagged message gets flagged.

Reply:
hmmm and I'm pretty sure `this.selectedUser.user` does not work `Envelope.vue` ;)

Reply:
recipient: this.recipient.email ?

Reply:
sounds right!